OpenRA.Mods.Common/Scripting/Global/EffectGlobal.cs

```diff
@@ -32,5 +35,17 @@ public void Flash(string type = null, int ticks = -1)
 				if (fpe.Info.Type == type)
 					fpe.Enable(ticks);
 		}
+
+		[Desc("Dynamically adjusts the `GlobalLightingPaletteEffect` trait.")]
+		public void ChangeLighting(double red, double green, double blue, double ambient)
```

Comment:
The trait use RGB. All WW ways to change lighting - map, scripts, superweapons - use RGB. Having this on HSL would be odd to me.